core/controller/src/main/scala/whisk/core/controller/Triggers.scala

```diff
@@ -321,6 +388,33 @@ trait WhiskTriggersApi extends WhiskCollectionAPI {
     complete(OK, trigger.withoutRules)
   }
 
+  /**
+   *
+   * @param triggerName
+   * @param ruleName
+   * @param actionName
+   * @param loglevel
+   * @param msg
+   * @return
+   */
+  private def triggerActivationLogMsg(triggerName: String,
```

Comment:
what's the point of this method?

Reply:
this method creates the trigger activation log entry string.  if we need the format to change, it's centralized here.